code/extensions/SiteTreeSubsites.php

```diff
@@ -208,10 +208,10 @@ static function contentcontrollerInit($controller) {
 		if($subsite && $subsite->Theme) SSViewer::set_theme(Subsite::currentSubsite()->Theme);
 	}
 
-	function alternateAbsoluteLink() {
+	function alternateAbsoluteLink($action) {
```

Comment:
An argument is passed through from the call in SiteTree (https://github.com/silverstripe/silverstripe-cms/blob/3.2/code/model/SiteTree.php#L477) but you are right, better to be safe.